### DISCUSSION TITLE
nvterm horizontal char/hl group

### DISCUSSION THREAD

**Discussion by ryanmsnyder** [NvChad/NvChad#2301] [Q&A] [answered]
Where is the horizontal fillchar being set? And how is it being highlighted in a way that doesn't affect nvimtree's window separator? 
<img width="721" alt="image" src="https://github.com/NvChad/NvChad/assets/41448219/877eb55f-ba58-47c5-af1a-5e350be2e591">

**Comment by siduck** [comment, marked as answer]
@ryanmsnyder not nvchad issue, its the issue with nvimntree probably

**Reply by ryanmsnyder** [reply]
@siduck there's no issue here. I'm just wondering how the separator for the terminal is being highlighted without affecting nvimtree's border. I thought any changes to `WinSeparator` affected all borders - both vertical and horizontal.

**Reply by tnypxl** [reply]
I know this is quite old now, but I'm dealing with this issue currently and I can't seem to figure out how to fix it.
